components/identity-core/org.wso2.carbon.identity.core/src/test/java/org/wso2/carbon/identity/core/util/IdentityUtilTest.java

```diff
@@ -811,4 +811,46 @@ public void testIsSupportedByUserStore() throws Exception {
     public IObjectFactory getObjectFactory() {
         return new org.powermock.modules.testng.PowerMockObjectFactory();
     }
+
+    private static void doBootstrap() {
```

Comment:
Can you please create a single method that can be used to bootstrap saml, rather than duplicating the code.

Reply:
Added a class named **SAMLInitializer** to do the initialization process of OpenSAML3 to avoid code duplication.